### DISCUSSION TITLE
Update steps for xpand when upgrading DevExpress from 16.1.8 to newest (.NET Framework compatible) version

### DISCUSSION THREAD

**Discussion by robertboc** [eXpandFramework/eXpand#1046] [Q&A]
Hello dear community,

in my enterprise I work on a windows application which extensively uses DevExpress as well as eXpand utilities.
For years the software components have not been upgraded in fear of breaking changes. I made it my duty to fix this.

What I'm concerned about are the update steps for eXpand that I need to follow for it to work. DevExpress support stated that it's best to upgrade step by step to the next major version until I reach my destination. Is it the same for the eXpandFramework, or does it suffice to integrate only the specific version that I eventually arrive at?
Do you have advice or best practices besides my question which you could link me to or enlighten me on?

Thank you so much for your answers.
Cheers, Robert

**Comment by apobekiaris** [comment]
Hello Robert,

eXpand versions as you probably know follow DevExpress and we have releases for each minor version. However 16.1.8 means that you should expect a lot of changes on both sides XAF and eXpand. We have this video on you you install the packages in order to avoid conflicts.

https://www.youtube.com/watch?v=LvxQ-U_0Sbg

Finally, if you want to speed up your work I am always available under paid contract to help you climp any mountain.

**Comment by robertboc** [comment]
Thank you very much for your quick answer and the link you provided. I'm aware that it'll be a project in itself to upgrade the software.
If anything else comes up I'm sure to drop by again here!
Cheers.
